model/project_ref.go

```diff
@@ -683,6 +683,9 @@ func (p *ProjectRef) getBatchTime(variant *BuildVariant) int {
 
 // return the next valid batch time
 func GetActivationTimeWithCron(curTime time.Time, cronBatchTime string) (time.Time, error) {
+	if strings.HasPrefix(cronBatchTime, "@every") {
+		return time.Time{}, errors.Errorf("cron batchtime '%s' cannot be an interval", cronBatchTime)
```

Comment:
nit, maybe `"cannot use '@every'"`, since `"cannot be an interval"` might not be clear to people